netbox/dcim/tables/template_code.py

```diff
@@ -20,19 +22,19 @@
 {% if value %}{{ value|floatformat:"-2" }} {{ record.weight_unit }}{% endif %}
 """
 
-DEVICE_LINK = """
+DEVICE_LINK = _("""
```

Comment:
Yes, but planning to address in separate issue as I think there needs to be some discussion around it and this makes them easier to find without any drawbacks and could be translated this way if needed.  

The issue is those strings contain template code and adding {% translate around the text wouldn't get picked up for the po files as they are in strings.  .format doesn't work as they contain template tags.  

Probably the correct solution is to move them out to actual templates (with the translation tags), change the referencing template_code= to template=.  However that begs the question should it only be done for the ones that need translation, or for all these table template strings...  I think probably all of them and create a sub-directory for them: table/fragments or something

Reply:
> Yes, but planning to address in separate issue as I think there needs to be some discussion around it and this makes them easier to find without any drawbacks and could be translated this way if needed.

Let's remove these from the PR. I'm not comfortable merging changes that we _know_ result in undesired behavior, even if we intend to fix it later.

> Probably the correct solution is to move them out to actual templates

IIRC the template code is defined like this because we encountered a performance issue when loading table column templates from file. That was years ago though and should be revisited.